backend/capellacollab/users/tokens/crud.py

```diff
@@ -25,7 +25,7 @@ def create_token(
     source: str,
 ) -> tuple[models.DatabaseUserToken, str]:
     password = "collabmanager_" + credentials.generate_password(32)
-    ph = argon2.PasswordHasher()
+    ph = argon2.PasswordHasher(time_cost=1, memory_cost=2048, parallelism=1)
```

Comment:
Our backend has a limit of 500MiB of memory, 2048 could OOM kill it.

Reply:
`memory_cost` is in kb, so this is 2MB which I don't think should be a problem. Running the CLI which tests hashing 100 passwords was also fine and didn't kill it.

Reply:
Ah, right.

Reply:
Also this was previously set to 65536, so reducing it to 2048 should not cause any issues